examples/iam-profile/README.md

```diff
@@ -0,0 +1,133 @@
+
+The following JSON profile grants all the permissions required to use
+all of the resources currently supported.
+
+## How
+
+Simple upload the following profile to your IAM account. It is included
+for clarity below, but you can download the [raw JSON
+file](profile.json) too.
+
+Note that as the number of resources the module supports grows we will
+add to this profile.
+
+```json
+{
+  "Version": "2015-02-13",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:DescribeInstances",
+        "ec2:RunInstances",
+        "ec2:StartInstances",
+        "ec2:StopInstances",
+        "ec2:DeleteTags",
+        "ec2:CreateTags",
+        "ec2:TerminateInstances",
+        "elasticloadbalancing:DescribeLoadBalancers",
+        "elasticloadbalancing:CreateLoadBalancer",
+        "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+        "elasticloadbalancing:DeleteLoadBalancer",
+        "ec2:DescribeSecurityGroups",
+        "ec2:CreateSecurityGroups",
+        "ec2:AuthorizeSecurityGroupIngress",
+        "ec2:RevokeSecurityGroupIngress",
+        "ec2:DeleteSecurityGroup",
+        "ec2:DescribeVpcs",
+        "ec2:CreateVpc",
+        "ec2:DeleteVpc",
+        "ec2:DescribeDhcpOptions",
+        "ec2:CreateDhcpOptions",
+        "ec2:DeleteDhcp_options",
+        "ec2:DescribeCustomerGateways",
+        "ec2:CreateCustomerGateway",
+        "ec2:DeleteCustomerGateway",
+        "ec2:DescribeInternetGateways",
+        "ec2:CreateInternetGateway",
+        "ec2:DeleteInternetGateway",
+        "ec2:DetachInternetGateway",
+        "ec2:DescribeRouteTables",
+        "ec2:CreateRouteTable",
+        "ec2:DeleteRouteTable",
+        "ec2:CreateRoute",
+        "ec2:DescribeSubnet",
+        "ec2:CreateSubnet",
+        "ec2:DeleteSubnet",
+        "ec2:AssociateRouteTable",
+        "ec2:DescribeVpnConnections",
+        "ec2:CreateVpnConnection",
+        "ec2:DeleteVpnConnection",
+        "ec2:CreateVpnConnectionRoute",
+        "ec2:DescribeVpnGateways",
+        "ec2:CreateVpnGateway",
+        "ec2:AttachVpnGateway",
+        "ec2:DetachVpnGateway",
+        "ec2:DeleteVpnGateway",
+        "ec2:CreateTags",
+        "autoscaling:DescribeAutoScalingGroups",
+        "autoscaling:CreateAutoScalingGroup",
+        "autoscaling:UpdateAutoScalingGroup",
+        "autoscaling:DeleteAutoScalingGroup",
+        "autoscaling:DescribeLaunchConfigurations",
+        "autoscaling:CreateLaunchConfiguration",
+        "autoscaling:DeleteLaunchConfiguration",
+        "autoscaling:DescribePolicies",
+        "autoscaling:PutScalingPolicy",
+        "autoscaling:DeletePolicy",
+        "cloudwatch:DescribeAlarms",
+        "cloudwatch:PutMetricAlarm",
+        "cloudwatch:DeleteAlarms",
+        "route53:ListResourceRecordSets",
+        "route53:ListHostedZones",
+        "route53:ChangeResourceRecordSets",
+        "route53:CreateHostedZone",
+        "route53:DeleteHostedZone"
+      ],
```

Comment:
If you haven't already, I would suggest creating a user with these perms in the Amazon account where we're admin, and running acceptance over top of it for validation.